src/reviewerQueue.js

```diff
@@ -109,25 +108,25 @@ HUBOT_GITHUB_REVIEWER_EMAIL_MAP: ${ghReviewerEmailMap}\
       stats.reviewers = reviewers;
     }
 
-    let reviewersOnVacation = {};
-    let reviewerEmailMap = JSON.parse(ghReviewerEmailMap);
-    for (let event of travelEvents) {
-      let reviewerLogin = reviewerEmailMap[event.creator.email];
-      if (event.summary.match(/(ooo|vacation)/i)) {
-        reviewersOnVacation[reviewerLogin] = true;
-        robot.logger.debug(`${reviewerLogin} is ooo`);
-      }
-    }
-
     // pick reviewer
     reviewers = stats.reviewers;
-    reviewers = reviewers.filter((r) => r.login !== creator.login && !reviewersOnVacation[r.login]);
-
+    reviewers = reviewers.filter((r) => r.login !== creator.login);
+    
     // exclude current assignee from reviewer candidates
     if (assignee !== null) {
       reviewers = reviewers.filter((r) => r.login !== assignee.login);
     }
 
+    // exclude shadows from reviewer candidates
+    let reviewer_shadows = new Set();
```

Comment:
nit here and elsewhere: Prefer camelCase like the rest of the file's variables.

Reply:
Done.